pkg/kwokctl/runtime/kind/kind.yaml.tpl

```
@@ -23,6 +23,12 @@ nodes:
     listenAddress: "0.0.0.0"
     protocol: TCP
   {{ end }}
+  {{ if .EtcdPort }}
+  - containerPort: {{ .EtcdPort }}
```

```suggestion
  - containerPort: 2379
```

The port of Etcd in Kind configuration is here, 

https://github.com/kubernetes-sigs/kwok/blob/cbe4878f0a7a41b77fca5c9840be976747c0fefc/pkg/kwokctl/runtime/kind/prometheus_deployment.yaml.tpl#L70